test/verify/check-multi-machine

```diff
@@ -859,6 +859,12 @@ class TestMultiMachine(MachineCase):
         b.wait_present("#edit-machine-port")
         b.set_val("#edit-machine-port", "2222")
         b.click('#troubleshoot-dialog .btn-primary')
+        # Using the libssh's knownhosts api port is taken into account when verifying a host
```

Comment:
I just confirmed that ssh (CLI) behaves the same way, so this is really a bug fix. This exceptional case in the test is a bit ugly, but it will reduce over time. But would it be possible and reasonably easy to implement port checking in our shim as well? (follow-up is fine)